pages/shows/index.vue

```diff
@@ -41,7 +41,7 @@ export default {
       return null
     },
     pastShowsList () {
-      if (this.arcsiShows.length > 0) {
+      if (this.arcsiShows) {
```

Comment:
Maybe use the optional chaining, eg. `this.arcsiShows?.length > 0` to check all states? I would leave the length, since it might also be empty if the API server hangs